src/main/java/uk/gov/gchq/palisade/exception/PalisadeCheckedException.java

```diff
@@ -15,6 +15,8 @@
  */
 package uk.gov.gchq.palisade.exception;
 
+import uk.gov.gchq.palisade.Generated;
+
 import static uk.gov.gchq.palisade.exception.Status.INTERNAL_SERVER_ERROR;
 
 /**
```

Comment:
Use the setters available for each of the methods below

Reply:
? The way that was reading this Exception and the others is that they should not be changed after they are created.  That means no setters.  Note I have not added equals, hashcode and toString so they will depend on Throwable for the toString.   The other methods will revert to Object.  This is the way that I found them.  It make sense if we would never want to collect errors in a single task.  Does that sound right?